scipy/interpolate/__init__.py

```diff
@@ -209,6 +210,8 @@
 
 from ._pade import *
 
+from ._regrid_python import regrid_python, ndbspline_call_like_bivariate
```

Comment:
It is not clear if `ndbspline_call_like_bivariate` is meant to be public as it hasn't been added to the toc tree but has been imported here

Reply:
User-facing interface might need some more thought. In my (somewhat rough) ideas, I think we should have a reasonable new function and a documentation guide for porting from RectBivariateSpline to the new framework. 
Then `ndbspline_call_like_bivariate` is needed internally for tests and its simplified version ends up in the docs as a recipe.

However, if we want to actually remove the Fortran implementations, we likely also want to replace RectBivariateSpline implementations with the new ones: keep the API exactly the same, and replace the internals (cf the ODR thread on Discourse).

By all means let's discuss the strategy (and I'm all ears for suggestions!)

However no matter how we play it, I don't see `ndbspline_call_like_bivariate` becoming a part of public SciPy API, really.

Reply:
> I don't see `ndbspline_call_like_bivariate` becoming a part of public SciPy API, really.

Makes sense. Will remove it from the public API.



> In my (somewhat rough) ideas, I think we should have a reasonable new function and a documentation guide for porting from RectBivariateSpline to the new framework.

Makes sense. What about adding an option in `RectBivariateSpline.__init__(..., use_fitpack=True)`. When we pass `RectBivariateSpline(..., use_fitpack=False)` then our `_regrid_python.regrid_python` API will be called. Also, in `RectBivariateSpline.__call__` we will check whether the object is `NdBSpline`. If so then we would call `ndbspline_call_like_bivariate`. So nothing will be exposed to the user. :-)

@ev-br What do you say?